### DISCUSSION TITLE
Idea - Move the Default Library addon to be included in the Essentials library compiled in by default

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#4092] [Ideas]
I found a way to edit and compile the default library called "Essentials". We have like 2 default libraries, one is registered by addon, the other is compiled in. But you can edit the source files before compiling them.

So I'm wondering, maybe we can include these files into the essentials library? 

Pro:
- One default library
- More discoverable and feels "built in"

Con:
- The essentials library can only append, and not be linked.......................
- Default library would no-longer have an opt-out option. 

**QUESTION** Is it worth combining? or should we keep the default library as an addon that could be opt-out?

**Comment by ghost** [comment]
let's keep it separate for now, as an addon, more modular and more benefits than built in library
